app/graphql/types/world_index_type.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Types
+  class WorldIndexType < Types::EditionType
+    field :international_delegations, [WorldLocationType]
+    field :world_locations, [WorldLocationType]
+
+    def self.base_path = "/world"
+
+    def international_delegations
+      simplify_world_locations(object.details[:international_delegations])
+    end
+
+    def world_locations
+      simplify_world_locations(object.details[:world_locations])
+    end
+
+  private
+
+    def simplify_world_locations(world_locations)
+      world_locations.map { |world_location| world_location.slice(:active, :name, :slug) }
+    end
```

Comment:
Do we need to do this step?  The current content item includes all the fields stored in the `details` for both world locations and international delegations, not just these three.